rxv/ssdp.py

```diff
@@ -62,7 +62,10 @@ def discover(timeout=1.5):
 def rxv_details(location):
     """Looks under given UPNP url, and checks if Yamaha amplituner lives there
        returns RxvDetails if yes, None otherwise"""
-    xml = ET.XML(requests.get(location).content)
+    try:
+        xml = ET.XML(requests.get(location).content)
+    except:
```

Comment:
This is the error I was encountering:

> requests.exceptions.ConnectionError: HTTPConnectionPool(host='x.x.x.x', port=8200): Max retries exceeded with url: /rootDesc.xml (Caused by <class 'socket.error'>: [Errno 54] Connection reset by peer)

I suppose I could change this to catch `requests.exceptions.ConnectionError`, but then if the HHTP request errors out for some other reason (e.g., 4xx or 5xx response code) or the request is fine but there's some kind of XML parse error we'd hit the same undesired behavior.

Generally speaking, I think catching exceptions here broadly is the right thing to do, since we're putting ourselves out there and hoping for a very specific result. Anything unexpected means that we're likely not talking to a Yamaha RXV* and we should just move on. That said, I don't know python all that well, so maybe it's better practice to catch a superclass of request errors and a general XML error instead of just everything?